A few new hails for friendly human merchants (#8739)
endless-sky/endless-sky: data/human/hails.txt

		"Zug. I used to watch the militia ships fight against the pirate ships in the sky from my porch."
		"New Holland. It gets too boring there, though, so I fly the stars instead."
		"Bounty. I bought my first ship when the manufacturing industry started to develop there."
		

> phrase "pyramid scheme"

Author: ```suggestion
phrase "pyramid_scheme_company"
```

	word
		"Magellan "
		"Dehydrated "
		"Pontificated "
		"General "

> 		"Sex Sex Sex "

Author: Presumably you're talking about line 2092? That's fair enough, I'm happy to remove it.

> 		"Sex Sex Sex "

Author: ```suggestion
```

Reviewer: This suggestion didn't get committed if you wanted it to be, it was only resolved.

> 		"Sex Sex Sex "

Reviewer 2: Yeah, I don't really see any benefit in being explicit like that. 

Perhaps "Sensual" or "Risque", perhaps?

		"Sacrosanct "
		"Liberated "
		"Literal "

> 		"Family "

Author: ```suggestion
		"Family "
		"Hyper-real "
```

	word
		"Dynamix"
		"Unlimited"
		"Orbital"

> 		"Hyper-real"

Author: ```suggestion
```

		"Gigalight"
		"Traders"
		"Voidcraft"
		"Whispers"

phrase "friendly civilian"
	word
		"Hi! Good to see you, uh, "
		"Well, if it isn't "
		"Hi there, uhm, "
		"I've been trying to get a hold of you, "
	word
		"Captain <Last>. "
	word
		"I'm living the dream as a CEO of my own company. "
		"I used to be just like you, but now I'm really somebody! "
		"Don't you want to be your own boss? "
		"Everybody is talking about this slick new fad! "
	word
		"You could get rich selling for me!"
		"Get in on the ground floor!"

> 		"We're about to go orbital, you should sign up while you still can!"

Reviewer: You're missing a space here.
```suggestion
		"We're about to go orbital, you should sign up while you still can!"
	word
		" "
```

	word
		"Find me "
		"You can get me "
		"Call me "
		"Drop me a line "
		"Buzz me "
	word

> 		"at ${name}@${pyramid scheme}."

Author: ```suggestion
		"at ${name}@${pyramid_scheme_company}."
```

> 		"at ${name}@${pyramid scheme}."

Author: ```suggestion
		"at ${name} at ${pyramid scheme}."
```

Reviewer: This suggestion too?

phrase "friendly civilian"
	word
		"All I want is one day without "
		"I just bought this ship and it already has "
		"The warranty for my ship expired a week ago, and wouldn't you know it, I just found "
	word
		"a fuel leak! "
		"a loose coupling! "
		"a fault in my nav-array! "

> 		"a power cascade! "

Author: That's what I had in mind, yeah.

> 		"a power cascade! "

Author: ```suggestion
		"a cascading failure in the power supply! "
```

> 		"an anti-matter containment failure! "

Author: I wasn't aware of antimatter being restricted in lore. I could change it to deuterium or something.

> 		"an anti-matter containment failure! "

Reviewer 3: Yup, let's change it to deuterium if you don't mind. Antimatter seems like the kind of thing that we would have mentioned if it exists - seems a little weird to just throw it in here.

> 		"an anti-matter containment failure! "

Reviewer 2: Currently, all human ships use some variation on fuel cell, fission, or fusion reactors, with solar still being an option. And some of those have descriptions that imply they are fairly recent versions/improvements. There's also no antimatter mentioned in use anywhere other than Quarg power supplies and Archon's main cannon. This suggests that humanity does not have significant antimatter production or usage, and that it is predominantly something that no-one outside of high end laboratories is going to have any exposure to.

> 		"an engine overheating! "

Reviewer: A few extra ones : )
```suggestion
		"an engine overheating! "
		"unsynchronized repulsors! "
		"a parasitic shield disharge! "
		"a leaky fuel cell! "
```

	word
		"The maintenance on this old bucket will cripple me."
		"It's just one thing after another on these old ships."
		"I wouldn't complain, but today is my birthday!"
		"And it's not as though parts are cheap these days."
		"It could be worse, I suppose. I could fly into a black hole."

> 		"It's not like the customers even care, they just want their cargo yesterday."

Reviewer: ```suggestion
		"It's not like the customers even care, they just want their cargo yesterday."
		"What else can you expect from shoddy Syndicated Systems manufacturing?"
		"I'd repair it, but I got to pay my mortgage."
		"Do you know where I could fix it for good?"
		"I'd love to know how other captains get by without these problems."
```

		
phrase "friendly civilian"
	word

> 		"I just found out that a 'duck' is a type of bird from Earth! Now I wonder why 'duck tape' is named after it?"
> 		"Apparently they want me to pay 5% of my paycheck for union dues! I'm gonna keep it. My boss is a good guy, he'll look out for me."
> 		"I saved up and bought extra bunk space so my wife could come on trips with me, and now she says 'we need to talk'. That can't be a good sign, can it?"

Reviewer 3: ```suggestion
		`I just found out that a "duck" is a type of bird from Earth! Now I wonder why "duck tape" is named after it?`
		`Apparently they want me to pay 5% of my paycheck for union dues! I'm gonna keep it. My boss is a good guy; he'll look out for me.`
		`I saved up and bought extra bunk space so my wife could come on trips with me, and now she says "we need to talk." That can't be a good sign, can it?`
```

A few grammar edits and changing to use the right type of quotation mark,

		"Pontificated "
		"General "
		"Sex Sex Sex "
		"Sacrosanct "
		"Liberated "
		"Literal "
		"Family "
	word
		"Dynamix"
		"Unlimited"
		"Orbital"
		"Hyper-real"
		"Gigalight"
		"Traders"
		"Voidcraft"
		"Whispers"

phrase "friendly civilian"
	word
		"Hi! Good to see you, uh, "
		"Well, if it isn't "
		"Hi there, uhm, "
		"I've been trying to get a hold of you, "
	word
		"Captain <Last>. "
	word
		"I'm living the dream as a CEO of my own company. "
		"I used to be just like you, but now I'm really somebody! "
		"Don't you want to be your own boss? "
		"Everybody is talking about this slick new fad! "
	word
		"You could get rich selling for me!"
		"Get in on the ground floor!"
		"We're about to go orbital, you should sign up while you still can!"
	word
		" "
	word
		"Find me "
		"You can get me "
		"Call me "
		"Drop me a line "
		"Buzz me "
	word
		"at ${name}@${pyramid scheme}."

phrase "friendly civilian"
	word
		"All I want is one day without "
		"I just bought this ship and it already has "
		"The warranty for my ship expired a week ago, and wouldn't you know it, I just found "
	word
		"a fuel leak! "
		"a loose coupling! "
		"a fault in my nav-array! "
		"a power cascade! "

> 		"an anti-matter containment failure! "

Author: ```suggestion
		"a deuterium containment failure! "
```

Reviewer 4: Deuterium isn't really dangerous unlike antimatter.

Reviewer: Could change it to oxygen. You got a hull breach and air leaking out of your ship, that's a bad sign.

		"Zug. I used to watch the militia ships fight against the pirate ships in the sky from my porch."
		"New Holland. It gets too boring there, though, so I fly the stars instead."
		"Bounty. I bought my first ship when the manufacturing industry started to develop there."
		

> phrase "pyramid_scheme_company"

Reviewer 5: Don't use underscores here.
```suggestion
phrase "pyramid scheme company"
```

	word
		"Magellan"
		"Dehydrated"
		"Pontificated"
		"General"
		"Sacrosanct"
		"Liberated"
		"Literal"
		"Family"
		"Hyper-real"
	word
		" "
	word
		"Dynamix"
		"Unlimited"
		"Orbital"
		"Gigalight"
		"Traders"
		"Voidcraft"
		"Whispers"

phrase "friendly civilian"

> 	word
> 		"Hi! Good to see you, uh, "
> 		"Well, if it isn't "
> 		"Hi there, uhm, "
> 		"I've been trying to get a hold of you, "

Reviewer 5: ```suggestion
	word
		"Hi! Good to see you, uh,"
		"Well, if it isn't"
		"Hi there, uhm,"
		"I've been trying to get a hold of you,"
```

	word

> 		"Captain <last>. "

Reviewer 5: ```suggestion
		" Captain <last>. "
```

> 	word
> 		"I'm living the dream as a CEO of my own company. "
> 		"I used to be just like you, but now I'm really somebody! "
> 		"Don't you want to be your own boss? "
> 		"Everybody is talking about this slick new trend! "

Reviewer 5: ```suggestion
	word
		"I'm living the dream as a CEO of my own company."
		"I used to be just like you, but now I'm really somebody!"
		"Don't you want to be your own boss?"
		"Everybody is talking about this slick new trend!"
	word
		" "
```

	word
		"You could get rich selling for me!"
		"Get in on the ground floor!"
		"We're about to go orbital, you should sign up while you still can!"
	word
		" "
	word

> 		"Find me "
> 		"You can get me "
> 		"Call me "
> 		"Drop me a line "
> 		"Buzz me "
> 	word
> 		"at ${pyramid_scheme_company}. "

Reviewer 5: ```suggestion
		"Find me"
		"You can get me"
		"Call me"
		"Drop me a line"
		"Buzz me"
	word
		" at ${pyramid scheme company}. "
```

	word
		"Ask for ${name}."
		"Tell them ${name} sent you."
		"Mention my name, ${name}, for a discount on your first order!"

phrase "friendly civilian"

> 	word
> 		"All I want is one day without running into "
> 		"I just bought this ship and it already has "
> 		"The warranty for my ship expired a week ago, and wouldn't you know it, I just found "

Reviewer 5: ```suggestion
	word
		"All I want is one day without running into"
		"I just bought this ship and it already has"
		"The warranty for my ship expired a week ago, and wouldn't you know it, I just found"
	word
		" "
```

> 	word
> 		"a fuel leak! "
> 		"a loose coupling! "
> 		"a fault in my nav-array! "
> 		"a cascading failure in the power supply! "
> 		"a severe oxygen leak! "
> 		"an engine overheating! "
> 		"unsynchronized repulsors! "
> 		"a parasitic shield discharge! "
> 		"a leaky fuel cell! "

Reviewer 5: ```suggestion
	word
		"a fuel leak!"
		"a loose coupling!"
		"a fault in my nav-array!"
		"a cascading failure in the power supply!"
		"a severe oxygen leak!"
		"an engine overheating!"
		"unsynchronized repulsors!"
		"a parasitic shield discharge!"
		"a leaky fuel cell!"
	word
		" "
```

		"New Holland. It gets too boring there, though, so I fly the stars instead."
		"Bounty. I bought my first ship when the manufacturing industry started to develop there."
		
phrase "pyramid_scheme_company"
	word
		"Magellan "
		"Dehydrated "
		"Pontificated "
		"General "
		"Sacrosanct "
		"Liberated "
		"Literal "
		"Family "
		"Hyper-real "
	word
		"Dynamix"
		"Unlimited"
		"Orbital"
		"Gigalight"
		"Traders"
		"Voidcraft"
		"Whispers"

phrase "friendly civilian"
	word
		"Hi! Good to see you, uh, "
		"Well, if it isn't "
		"Hi there, uhm, "
		"I've been trying to get a hold of you, "
	word
		"Captain <Last>. "
	word
		"I'm living the dream as a CEO of my own company. "
		"I used to be just like you, but now I'm really somebody! "
		"Don't you want to be your own boss? "
		"Everybody is talking about this slick new fad! "
	word
		"You could get rich selling for me!"
		"Get in on the ground floor!"
		"We're about to go orbital, you should sign up while you still can!"
	word
		" "
	word
		" "
	word
		" "
	word
		"Find me "
		"You can get me "
		"Call me "
		"Drop me a line "
		"Buzz me "
	word

> 		"at ${name}@${pyramid_scheme_company}."

Author: ```suggestion
		"at ${pyramid_scheme_company}. "
	word
		"Ask for ${name}."
		"Tell them ${name} sent you."
		"Mention my name, ${name}, for a discount on your first order!"
```

Reviewer: Remember to hit the commit suggestions button : )

		"Zug. I used to watch the militia ships fight against the pirate ships in the sky from my porch."
		"New Holland. It gets too boring there, though, so I fly the stars instead."
		"Bounty. I bought my first ship when the manufacturing industry started to develop there."
		
phrase "pyramid_scheme_company"
	word

> 		"Magellan "
> 		"Dehydrated "
> 		"Pontificated "
> 		"General "
> 		"Sacrosanct "
> 		"Liberated "
> 		"Literal "
> 		"Family "
> 		"Hyper-real "

Reviewer 5: ```suggestion
		"Magellan"
		"Dehydrated"
		"Pontificated"
		"General"
		"Sacrosanct"
		"Liberated"
		"Literal"
		"Family"
		"Hyper-real"
	word
		" "
```

	word
		"Dynamix"
		"Unlimited"
		"Orbital"
		"Gigalight"
		"Traders"
		"Voidcraft"

> 		"Whispers"

Author: I've got to admit, I've looked over this a few times and am still a bit lost. How does adding the third word allow us to make an address?

phrase "friendly civilian"
	word
		"Hi! Good to see you, uh, "
		"Well, if it isn't "
		"Hi there, uhm, "
		"I've been trying to get a hold of you, "
	word

> 		"Captain <Last>. "

Reviewer 5: How do they know our name?

Author: I had assumed that your IFF transponder would have that kind of basic information.

Reviewer 6: ```suggestion
		"Captain <last>. "
```
Needs to be lowercase.

	word
		"I'm living the dream as a CEO of my own company. "
		"I used to be just like you, but now I'm really somebody! "
		"Don't you want to be your own boss? "

> 		"Everybody is talking about this slick new fad! "

Reviewer 5: "Fad" is almost entirely negative.
```suggestion
		"Everybody is talking about this slick new trend! "
```

	word
		"You could get rich selling for me!"
		"Get in on the ground floor!"
		"We're about to go orbital, you should sign up while you still can!"
	word
		" "

> 	word
> 		" "
> 	word
> 		" "

Reviewer 5: ```suggestion
```

	word
		"Find me "
		"You can get me "
		"Call me "
		"Drop me a line "
		"Buzz me "
	word

> 		"at ${name}@${pyramid_scheme_company}."

Reviewer: Right now, this phrase might generate something like:

`Well, if it isn't Captain Last. I used to be just like you, but now I'm really somebody! We're about to go orbital, you should sign up while you still can!   Find me at James@General Dynamix.`

Do we want the three spaces between `still can!` and `Find me`, and the space in `General Dynamix`? `pyramid_scheme_company` is just the name of the phrase, so underscores don't make `General Dynamix` into `general_dynamix`. Plus, we don't know how the internet works in the 31st century, although they still use TCP/IP... but we will set a precedent for this sort of email address with this hail.

Author: Using your example, how about;

`Find me at General Dynamix, ask for James.`

> 		"Find me "
> 		"You can get me "
> 		"Call me "
> 		"Drop me a line "
> 		"Buzz me "
> 	word
> 		"at ${name}@${pyramid_scheme_company}."

Reviewer: ```suggestion
		"Find me"
		"You can get me"
		"Call me"
		"Drop me a line"
		"Buzz me"
		"Catch me"
		"Send me a message"
		"Join our team"
		"Join our family"
		"Get in touch with us"
		"Find out more from me"
		"You can find out more"
		"You'll want to get me"
	word
		" at "
	word
		"${name}@"
		"sales@"
		"joinus@"
		"applications@"
		"careers@"
		"outreach@"
		"onboarding@"
		"opportunities@"
	word
		"${pyramid scheme company 1}${pyramid scheme company 2}"
	word
		"!"
		"."
	word
		" Don't miss this opportunity!"
		" You don't want to miss out!"
		"" 8
```

	word
		"Ask for ${name}."
		"Tell them ${name} sent you."
		"Mention my name, ${name}, for a discount on your first order!"

phrase "friendly civilian"
	word

> 		"All I want is one day without "

Reviewer 5: Making this more grammatically consistent with the other options.
```suggestion
		"All I want is one day without running into "
```

		"I just bought this ship and it already has "
		"The warranty for my ship expired a week ago, and wouldn't you know it, I just found "
	word
		"a fuel leak! "
		"a loose coupling! "
		"a fault in my nav-array! "
		"a cascading failure in the power supply! "
		"a severe oxygen leak! "
		"an engine overheating! "
		"unsynchronized repulsors! "
		"a parasitic shield disharge! "
		"a leaky fuel cell! "
		"unsynchronized repulsors! "

> 		"a parasitic shield disharge! "

Reviewer 7: ```suggestion
		"a parasitic shield discharge! "
```

> 		"unsynchronized repulsors! "
> 		"a parasitic shield disharge! "
> 		"a leaky fuel cell! "
> 		"unsynchronized repulsors! "
> 		"a parasitic shield disharge! "
> 		"a leaky fuel cell! "

Reviewer 5: ```suggestion
		"unsynchronized repulsors! "
		"a parasitic shield discharge! "
		"a leaky fuel cell! "
```

	word
		"The maintenance on this old bucket will cripple me."
		"It's just one thing after another on these old ships."
		"I wouldn't complain, but today is my birthday!"
		"And it's not as though parts are cheap these days."
		"It could be worse, I suppose. I could fly into a black hole."
		"It's not like the customers even care, they just want their cargo yesterday."

> 		"What else can you expect from shoddy Syndicated Systems manufacturing?"

Reviewer 7: The ship that the hailing merchant has won't necessarily be a Syndicate one.

Reviewer: Not necessarily, but most ships will have some sort of Syndicate subsystems. I'm thinking doors, oxygen, life support, hyperdrives, cybernetics/ship controls, data panels, etc. That's why the Free Worlds needs the Syndicate for their ship manufacturing.
From Hephaestus:
> Syndicated Systems, which manufactures parts used in shipyards throughout human space

		"I'd repair it, but I got to pay my mortgage."
		"Do you know where I could fix it for good?"
		"I'd love to know how other captains get by without these problems."
		

> phrase "friendly civilian"

Reviewer 5: Avoid having phrases with only a few possible unique word options; they end up showing up a disproportionate amount.

Author: Yeah, I was thinking that I could weight these so that they show up less often, but I'm not sure of the best weighting to achieve that.

Reviewer 5: Put them as spaceport news instead.

> 		"What else can you expect from shoddy Syndicated Systems manufacturing?"
> 		"I'd repair it, but I got to pay my mortgage."
> 		"Do you know where I could fix it for good?"
> 		"I'd love to know how other captains get by without these problems."

Reviewer: ```suggestion
```
Oops, looks like this got pushed twice.
